Nitrox.Assets.Subnautica/LanguageFiles/en.json

```diff
@@ -1,69 +1,71 @@
+  "Nitrox_DiscordPressToFocus": "Press to unlock",
+  "Nitrox_DiscordRequestText": "Requested to join your session",
+  "Nitrox_EnterName": "Enter your player name",
+  "Nitrox_Failure": "An error occured",
+  "Nitrox_FirewallInterfering": "Seems like your firewall settings are interfering",
+  "Nitrox_HideIp": "Hide IP addresses",
+  "Nitrox_HidePing": "Hide Ping",
+  "Nitrox_InvalidUserName": "Please enter a valid player name.\n\nIt can not contain any space or dubious characters\nAllowed characters: A-Z a-z 0-9 _ . -\nLength: [3, 25]",
+  "Nitrox_Join": "Join",
+  "Nitrox_JoinServerFailed": "Unable to join server",
+  "Nitrox_JoinServerIpAddress": "Server IP",
+  "Nitrox_JoinServerPassword": "Password:",
+  "Nitrox_JoinServerPasswordHeader": "Server password required",
+  "Nitrox_JoiningSession": "Joining Multiplayer Session",
+  "Nitrox_Kick": "Kick {PLAYER}",
+  "Nitrox_KickQuestion": "Kick {PLAYER}?",
+  "Nitrox_LaunchGame": "Launching game…",
+  "Nitrox_LoadingScreenWarn": "Expect game-breaking bugs",
+  "Nitrox_LostConnection": "Lost connection to game server",
+  "Nitrox_LostConnectionWarn": "No longer connected to the server",
+  "Nitrox_Multiplayer": "Multiplayer",
+  "Nitrox_MultiplayerLoaded": "Multiplayer client loaded…",
+  "Nitrox_Mute": "Mute {PLAYER}",
+  "Nitrox_MuteQuestion": "Mute {PLAYER}?",
+  "Nitrox_MutedPlayer": "Muted {PLAYER}",
+  "Nitrox_PlayerColor": "Player Color",
+  "Nitrox_PlayerKicked": "You've been kicked from the server",
+  "Nitrox_PlayerListTabName": "Player List",
+  "Nitrox_RejectedSessionPolicy": "Reservation rejected…",
+  "Nitrox_RequestingSessionPolicy": "Requesting session policy info…",
+  "Nitrox_ServerStopped": "The server has been stopped",
+  "Nitrox_ShowPing": "Show Ping",
+  "Nitrox_SilenceChat": "Mute chat",
+  "Nitrox_SilencedChatNotif": "Chat is now muted",
+  "Nitrox_StartServer": "Start your server first to join your self-hosted world",
+  "Nitrox_StreamerSettings": "Streamer mode",
+  "Nitrox_SubmitPassword": "Submit",
+  "Nitrox_SyncingWorld": "Syncing multiplayer world…",
+  "Nitrox_TeleportTo": "Teleport to {PLAYER}",
+  "Nitrox_TeleportToMe": "Teleport {PLAYER} to me",
+  "Nitrox_TeleportToMeQuestion": "Teleport {PLAYER} to me ?",
+  "Nitrox_TeleportToQuestion": "Teleport to {PLAYER}?",
+  "Nitrox_ThankForPlaying": "Thanks for using Nitrox!",
+  "Nitrox_UnableToConnect": "Unable to contact the remote server at:",
+  "Nitrox_Unmute": "Unmute {PLAYER}",
+  "Nitrox_UnmuteQuestion": "Unmute {PLAYER}?",
+  "Nitrox_UnmutedPlayer": "Unmuted {PLAYER}",
+  "Nitrox_Waiting": "Waiting in join queue…",
+  "Nitrox_WaitingPassword": "Waiting for server password input…",
+  "Nitrox_WaitingUserInput": "Waiting for user input…",
+  "Nitrox_WorldSettling": "Awaiting World Settling…",
+  "Nitrox_RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target.",
+  "Nitrox_PlayerJoined": "joined the game.",
+  "Nitrox_PlayerLeft": "left the game."
```

Comment:
Maybe we can use the same system as Nitrox_Unmute with the {PLAYER} placeholder, so it's understandable.